client/src/app/metadata-schema-form/form-item/form-item.component.ts

```diff
@@ -0,0 +1,35 @@
+import {Component, Input} from "@angular/core";
+
+export interface FormItemData {
+
+  label: string;
+  helperText: string;
+  disabled: boolean;
+  isRequired: boolean;
+
+}
+
+@Component({
+  selector: 'form-item',
+  templateUrl: './form-item.component.html',
+  styleUrls: ['./form-item.component.css']
+})
+export class FormItemComponent {
+
+  @Input()
+  data: FormItemData;
+
+  private _rootClass = <any> {
+    "vf-form__item": true
+  }
+
+  @Input()
+  get rootClass(): any {
+    return this._rootClass;
+  }
+
+  set rootClass(rootClass: any) {
+    Object.assign(this._rootClass, rootClass);
+  }
+
```

Comment:
why not just create a rootClass var and annotate it with Input?

```
@Input()
rootClass
```

Reply:
Because there's a default class that gets merged to any additional class.